### PR TITLE
feat: intmdb 954 project enable extended storages

### PR DESCRIPTION
**Pull request by govardhanpagidi**
## Proposed changes

_Jira ticket: [INTMDB-954](https://jira.mongodb.org/browse/INTMDB-954)

- Adding IsExtendedStorageSizesEnabled property in the project settings API.
- Added missing descriptions for the Properties of Project Resource

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)
- [ ] This change requires a documentation update

## Manual QA performed:

- [ ] cfn invoke for each of CRUDL/cfn test
- [ ] Updated resource in  [example](https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/examples)
- [ ] Published to AWS private registry
- [ ] Used the template in [example](https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/examples) to create and update a stack in AWS
- [ ] Deleted stack to ensure resources are deleted
- [ ] Created multiple resources in same stack
- [ ] Validated in Atlas UI
- [ ] Included screenshots

## Required Checklist:

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation (if appropriate)
- [ ] I have run `make fmt` and formatted my code
- [ ] For CFN Resources: I have released by changes in the private registry and proved by change
  works in Atlas

### CODE REVIEW

**Comment by Adelmar92** [comment]
@govardhanpagidi can you add a description to the pull request, and the link to the jira ticket?

File: cfn-resources/project/mongodb-atlas-project.json [hunk 1]
```diff
@@ -19,6 +19,9 @@
                 },
                 "IsSchemaAdvisorEnabled": {
                     "type": "boolean"
+                },
+                "IsExtendedStorageSizesEnabled": {
+                    "type": "boolean"
```
Comment (Adelmar92) [inline cfn-resources/project/mongodb-atlas-project.json:24]: this new property is just to create?? does it need to be read only??

Comment (govardhanpagidi) [reply cfn-resources/project/mongodb-atlas-project.json]: IsExtendedStorageSizesEnabled is updatable property.

**Comment by govardhanpagidi** [comment]
> @govardhanpagidi can you add a description to the pull request, and the link to the jira ticket?

Updated the descriptions

The PR is already linked to the JIRA Ticket

image

**Review by Adelmar92 — APPROVED** [review]

**Pull request merged**